Content.Server/Construction/Completions/SetAnchor.cs

```diff
@@ -0,0 +1,28 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+using Content.Shared.Construction;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.Construction.Completions
+{
+    [UsedImplicitly]
+    public class SetAnchor : IGraphAction
+    {
+        public void ExposeData(ObjectSerializer serializer)
+        {
+            serializer.DataField(this, x => x.Value, "value", true);
+        }
+
+        public bool Value { get; private set; } = false;
```

Comment:
I'll just set it to true here. This needs to default to true, imo. (Because of the name SetAnchor, kinda implies that it sets the anchor to true if you don't pass arguments, in my opinion)